lib/parse.js

```diff
@@ -1,54 +1,47 @@
 "use strict";
 
 const babylonToEspree = require("./babylon-to-espree");
-const parse = require("@babel/parser").parse;
+const { parseSync: parse, traverse } = require("@babel/core");
 const tt = require("@babel/parser").tokTypes;
```

Comment:
@loganfsmyth You had mentioned [exporting `@babel/parser` in `@babel/core`](https://github.com/babel/babel-eslint/pull/594#discussion_r169840591). This makes a lot of sense to me, and it would allow us to only have `@babel/core` as a dependency.

Reply:
That would be awesome, and would remove the need to import it [here](https://github.com/airbnb/babel-plugin-inline-react-svg/blob/master/src/index.js#L3), which would have allowed simultaneous babel 6 and 7 compat, and may allow for simultaneous 7 and 8+ compact moving forwards.

Reply:
@kaicataldo The issue with doing this as-is, and the reason I think the other PR is important, is that there isn't any guarantee that the token typess returned from `parseSync` will actually be the same as `tokTypes` here. If the packages aren't deduped, they could be totally independent objects that are just deep-equal. The only way to be confident that the `tokTypes` values will match is to re-export them.

Reply:
Thanks for the explanation! That makes sense. Will update my other PR.